### PR TITLE
Rewrite TestNG Assertions to AssertJ

### PR DESCRIPTION
**Pull request by ssheikin**
Changed automatically by openrewrite recipe
https://github.com/openrewrite/rewrite-testing-frameworks/pull/520
```
<plugins>
    <plugin>
        <groupId>org.openrewrite.maven</groupId>
        <artifactId>rewrite-maven-plugin</artifactId>
        <version>5.32.0</version>
        <configuration>
            <activeRecipes>
                <recipe>org.openrewrite.java.testing.assertj.TestNgToAssertj</recipe>
            </activeRecipes>
        </configuration>
        <dependencies>
            <dependency>
                <groupId>org.openrewrite.recipe</groupId>
                <artifactId>rewrite-testing-frameworks</artifactId>
                <version>2.11.0-SNAPSHOT</version>
            </dependency>
        </dependencies>
    </plugin>
</plugins>
```

run
```
./mvnw rewrite:run -Dmaven.javadoc.skip=true -DskipTests=true -Dmaven
.site.skip=true -Dmaven.artifact.threads=16 -T 1C -e -Dair.check.skip-all=true --no-snapshot-updates
```

### CODE REVIEW

**Comment by wendigo** [comment]
Doesn't compile. Please fix compilation

**Comment by wendigo** [comment]
Nice!

**Comment by wendigo** [comment]
Reviewed and discussed offline.

**Comment by dain** [comment]
Nice!

File: jaxrs/src/test/java/io/airlift/jaxrs/TestJsonMapper.java [hunk 1]
```diff
@@ -167,7 +166,7 @@ public int read(byte[] b, int off, int len)
             fail("Should have thrown an IOException");
         }
         catch (WebApplicationException e) {
-            fail("Should not have received a WebApplicationException", e);
+            org.assertj.core.api.Assertions.fail("Should not have received a WebApplicationException", e);
```
Comment (findepi) [inline jaxrs/src/test/java/io/airlift/jaxrs/TestJsonMapper.java:169]: nit: maybe can be imported statically

**Comment by findepi** [comment]
@ssheikin awesome

**Pull request merged**